chronograf.go

```diff
@@ -168,14 +256,68 @@ type Template struct {
 
 // Query retrieves a Response from a TimeSeries.
 type Query struct {
-	Command      string        `json:"query"`              // Command is the query itself
-	DB           string        `json:"db,omitempty"`       // DB is optional and if empty will not be used.
-	RP           string        `json:"rp,omitempty"`       // RP is a retention policy and optional; if empty will not be used.
-	TemplateVars []TemplateVar `json:"tempVars,omitempty"` // TemplateVars are template variables to replace within an InfluxQL query
-	Wheres       []string      `json:"wheres,omitempty"`   // Wheres restricts the query to certain attributes
-	GroupBys     []string      `json:"groupbys,omitempty"` // GroupBys collate the query by these tags
-	Label        string        `json:"label,omitempty"`    // Label is the Y-Axis label for the data
-	Range        *Range        `json:"range,omitempty"`    // Range is the default Y-Axis range for the data
+	Command      string       `json:"query"`              // Command is the query itself
+	DB           string       `json:"db,omitempty"`       // DB is optional and if empty will not be used.
+	RP           string       `json:"rp,omitempty"`       // RP is a retention policy and optional; if empty will not be used.
+	TemplateVars TemplateVars `json:"tempVars,omitempty"` // TemplateVars are template variables to replace within an InfluxQL query
+	Wheres       []string     `json:"wheres,omitempty"`   // Wheres restricts the query to certain attributes
+	GroupBys     []string     `json:"groupbys,omitempty"` // GroupBys collate the query by these tags
+	Resolution   uint         `json:"resolution,omitempty"`
```

Comment:
Should mention in a comment that this is basically maximum resolution for the chart